modules/kubernetes/pages/service-certs.adoc

```diff
@@ -0,0 +1,140 @@
+// Type: assembly
+[id="encrypting-traffic-pod-router"] 
+= Encrypting traffic from a pod to the Skupper router
+
+This section describes how to apply certificates to encrypt the traffic within a cluster.
+
+By default:
+
+* {Skupper-name} creates certificates to establish links between sites using mutual TLS.
+This ensures that traffic between sites is encrypted. To use custom certificates for traffic between sites, see xref:index.adoc[].
+
+* Network traffic between pods and the Skupper router is not encrypted. 
+
+You can use certificates to encrypt traffic between pods and the Skupper router as follows:
+
+* xref:skupper-generated-certs[] - does not require user to provide certificates.
+* xref:user-provided-certs[] - requires that the certificate authority is distributed across all sites.
+
+NOTE: With both procedures, the certificates are provided when you expose the service on the {service-network}.
+
+// Type: procedure
+[id="skupper-generated-certs"] 
+= Exposing services on the {service-network} with Skupper-generated certificates
+
+A {skupper-name} installation includes a certificate authority which can generate certificates that can be used to encrypt traffic from the pod to the Skupper router.
```

Comment:
the skupper CA is stored in a secret called `skupper-service-client` that will necessary to configure in components included in the VAN that will send request to a service exposed over TLS.

Reply:
do we need that detail here?

Reply:
Maybe not here, but it would be helpful to mention it in the same place where `skupper-tls-<service-name>` is; because the user has to be aware of those names to modify the deployments with those certs.